src/flows/shared/Resizer.tsx

```diff
@@ -157,7 +170,11 @@ const Resizer: FC<Props> = ({
   let body = children
 
   if (error) {
-    body = <div className="panel-resizer--error">{error}</div>
+    body = (
+      <div className="panel-resizer--error">
+        {makeErrorHumanFriendly(error, selectedFunction.name)}
```

Comment:
should we make this actionable like the other aggregate type notification we return in the data explorer? Like, rather than telling the user what to do - shouldn't we render a bottom for them to click that'll do that for them?

Reply:
That is a great idea. I'll add that in